### PR TITLE
[15.0][MIG] stock_lot_filter_available: Migration to version 15.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT36554

please @carlosdauden @victoralmau review this

### CODE REVIEW

File: stock_lot_filter_available/tests/test_stock_lot_filter_available.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+class TestStockLotFilterAvailable(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.StockProductionLot = cls.env["stock.production.lot"]
+        cls.company = cls.env.ref("base.main_company")
+        cls.loc = cls.env.ref("stock.stock_location_stock")
+        cls.product = cls.env["product.product"].create(
+            {"name": "Test product", "type": "product", "tracking": "lot"}
```
Comment (victoralmau) [inline stock_lot_filter_available/tests/test_stock_lot_filter_available.py]:
```suggestion
            {"name": "Test product", "detailed_type": "product", "tracking": "lot"}
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

**Comment by CarlosRoca13** [comment]
@victoralmau Changed!

**Review by victoralmau — APPROVED** [review]
Code and functionality OK

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**